search-commons/src/test/java/no/unit/nva/search/IndexingClientTest.java

```diff
@@ -116,21 +111,34 @@ void shouldNotThrowExceptionWhenTryingToDeleteNonExistingDocument() throws IOExc
         assertDoesNotThrow(() -> indexingClient.removeDocumentFromIndex("1234"));
     }
 
-    @Test
-    void addDocumentToIndex() throws IOException, SearchException {
-        UpdateResponse updateResponse = mock(UpdateResponse.class);
-        IndexDocument mockDocument = mock(IndexDocument.class);
-        when(mockDocument.toJsonString()).thenReturn("{}");
-        when(mockDocument.getDocumentIdentifier()).thenReturn(SortableIdentifier.next().toString());
-        RestHighLevelClientWrapper restHighLevelClient = mock(RestHighLevelClientWrapper.class);
-        when(restHighLevelClient.update(any(), any())).thenReturn(updateResponse);
-        IndexingClient indexingClient = new IndexingClient(restHighLevelClient);
-        indexingClient.addDocumentToIndex(mockDocument);
+    private RestHighLevelClientWrapper setupMockEsClient() throws IOException {
```

Comment:
Remove unused method toIndexDocument

Reply:
method toIndexDocument in line 136 is used in line 69.